sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -519,6 +517,8 @@ export class EventProcessor {
     } finally {
       log.eventProcessor(`[${this._id}] EventProcessor stopped.`);
     }
+
+    await this.abandonPartitionOwnerships();
```

Comment:
I think it makes sense to leave it as is, mostly because the context from the underlying error (in this case storage) will help them understand _why_ it's broken.